tests/test_YamlModel.py

```diff
@@ -268,6 +269,21 @@ def test_valid_model(self):
 
         model.validate_model()
 
+    def test_minus_in_formula(self):
+        """Test writing a model for a right hand side starting with a minus."""
+        model = YamlModel()
+
+        model.add_ode(state_id='x',
+                      right_hand_side='-x',
+                      initial_value=1)
+
+        model.validate_model()
+
+        yaml_dir = os.path.join(self.test_dir, 'test_minus_sign.yaml')
+        model.write_to_yaml(yaml_dir)
+
+        validate_yaml(yaml_dir)
```

Comment:
wait ... shouldn't this fail somehow?

Reply:
no, I wanted to test if things are working for the Model Editor (which should be able to deal with minus signs here...) :)